README.md

```diff
@@ -1,112 +1,137 @@
-# preCICE example for FSI: 1D elastic tube
+---
+title: 1D Elastic Tube
+permalink: unknown
+keywords: OpenFOAM, python
+summary: The 1D Elastic Tube is a FSI case, that consists of an internal flow in a flexible tube. The flow is unsteady and incompressible. This tutorial can be run using C++ or python.  Running the simulation takes a few minutes. 
+---
 
-## First Step: Choose a version
 
-This tutorial comes in two distinct versions, one written in C++ and one in Python. The code for each are located in the folders `cxx` and `python` respectively.
+## Setup
 
-Choose a version, then navigate down to the corresponding sections for [Python](#python-version) and [C++](#c-version) instructions.
+ALTERNATIVE TO THE LINK:
 
----
-## Python version
+We want to simulate the internal flow in a flexible tube as shown in the figure below (image from [1]).
 
-This version is realized using the Python API for preCICE. Check [this entry in the preCICE wiki](https://github.com/precice/precice/wiki/1D-elastic-tube-using-the-Python-API) for more information on this example.
+![FSI3 setup](images/setup.png)
 
-### Requirements
+Fluid is assumed to be incompressible flow and gravity is neglected. Due to the axisymmetry, the flow can be described using a quasi-two dimensional continuity and momentum equations. The motivation and exact formulation of the equations that we consider can be found in [2]. 
 
-- [preCICE](https://github.com/precice/precice/wiki/Get-preCICE)
+The following parameters have been chosen:
+- Length of the tube: L = 10
+- Inlet velocity: v_inlet = 10 + 3 sin (10 PI t)
```

Comment:
I think LaTeX also works:
```suggestion
- Inlet velocity: $$ v_inlet = 10 + 3 sin (10 PI t) $$
```
See https://www.precice.org/docs-meta-cheatsheet.html#latex-math-syntax

Reply:
actually, the latex thingie it's not working for me, you can see the $$ in the readme...

Reply:
I just hope it will appear correctly in the website. If not, we change it back later.